getting-started-tutorial/socless-tutorial/.gitignore

```diff
@@ -0,0 +1,33 @@
+# Distribution / packaging
```

Comment:
multiple .gitignores in this repo, not sure if this is desired

Reply:
This is cool for now. In case folks want to push each folder into their git repo